dietpi/dietpi-software

```diff
@@ -533,22 +540,135 @@ INDEX_WEBSERVER_TARGET=${INDEX_WEBSERVER_TARGET:=-2}
 		#--------------------------------------------------------------
 		#Install Software
 
-		#Installs that require DESKTOP_LXDE
+		#Installs that require a DESKTOP
 		if (( $TIGHTVNCSERVER == 1 )); then
-			DESKTOP_LXDE=1
+			if (( $DESKTOP_LXDE == 0 )) && (( $DESKTOP_MATE == 0 )) && (( $DESKTOP_GNUSTEP == 0 )); then
```

Comment:
I think we might need a "desktop preference system"? similar to the webserver one. I can add this in if you think its worth doing?

Also, when software is installed, the value is ```2```, when software is "to be installed" (selected for install), the value is ```1```. 
So when checking for both installed, and tobe installed, you can use ```if (( $MYPROGRAM > 0 )); then```